batch/test/test_database.py

```diff
@@ -17,9 +17,9 @@ def temp_table(self):
         t = self.db.create_temporary_table_sync("temp", schema, keys)
 
         for i in range(1, 6):
-            id = run_synchronous(t.new_record({'id': i,
-                                                'less_than_two': (i < 2),
-                                                'name': f"{i}"}))
+            id = run_synchronous(t.new_record(**{'id': i,
+                                                 'less_than_two': (i < 2),
+                                                 'name': f"{i}"}))
```

Comment:
follow up PR should convert these tests to use keyword syntax. Keyword syntax is an improvement! I don't want to leave cruft like this even if its just the tests.

Reply:
I changed these except the one that has a special character (back tick) in the field name.